BookReader/plugins/plugin.vendor-fullscreen.js

```diff
@@ -29,6 +34,12 @@
   })(BR.prototype.getInitialMode);
 
   BR.prototype.init = (function(super_) {
+    if (isMobile()) {
+      for (var method in super_props) {
```

Comment:
QUESTION: Where is `super_props` introduced?

Reply:
Whoops! Should be super_methods